README.md

```diff
@@ -34,14 +34,18 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
```

Comment:
Sematic code is very important here is a link with the most relevant reason why -> https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML